src/newcharacter.cpp

```diff
@@ -2177,19 +2180,26 @@ tab_direction set_description( const catacurses::window &w, avatar &you, const b
 
     uilist select_location;
     select_location.text = _( "Select a starting location." );
-    int offset = 0;
+    int offset = 1;
+    uilist_entry entry_random_start_location( RANDOM_START_LOCATION_ENTRY, true, -1,
```

Comment:
Considering that `RANDOM_START_LOCATION_ENTRY` is ony used in this function, you should just make it a local variable.
```C++
    const std::string RANDOM_START_LOCATION_TEXT = _( "<color_red>* Random location *</color>" );
```